jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/CronJobHandlerTest.java

```diff
@@ -0,0 +1,168 @@
+        .controllerName("testing")
+        .serviceAccount("test-account")
+        .restartPolicy("Never")
+        .volumes(volumes)
+        .build();
+
+    // When
+    CronJob cronJob = cronJobHandler.get(config, images);
+    // Then
+    assertThat(cronJob.getSpec().getJobTemplate().getSpec().getTemplate().getSpec().getContainers())
+        .isNotNull();
+    assertThat(cronJob)
+        .hasFieldOrPropertyWithValue("metadata.name", "testing")
+        .hasFieldOrPropertyWithValue("spec.schedule", "* * * * *")
+        .satisfies(cj -> assertThat(cj.getSpec().getJobTemplate().getSpec())
+            .isNotNull()
+            .extracting(JobSpec::getTemplate)
+            .extracting(PodTemplateSpec::getSpec)
+            .hasFieldOrPropertyWithValue("serviceAccountName", "test-account")
+            .hasFieldOrPropertyWithValue("restartPolicy", "Never")
+            .extracting(PodSpec::getVolumes).asList().first()
+            .hasFieldOrPropertyWithValue("name", "test")
+            .hasFieldOrPropertyWithValue("hostPath.path", "/test/path")
+        );
+  }
+
+  @Test
+  void get_withoutSchedule_shouldThrowException() {
+    // Given
+    ResourceConfig config = ResourceConfig.builder()
+        .controllerName("testing")
+        .serviceAccount("test-account")
+        .restartPolicy("Never")
+        .volumes(volumes)
+        .build();
+
+    // When & Then
+    assertThatIllegalArgumentException()
+        .isThrownBy(() -> cronJobHandler.get(config, images))
+        .withMessage("No schedule is specified!");
+  }
+
+  @Test
+  void get_withInvalidControllerName_shouldThrowException() {
+    // Given
+    ResourceConfig config = ResourceConfig.builder()
+        .schedule("* * * * *")
+        .imagePullPolicy("IfNotPresent")
+        .controllerName("TesTing")
+        .serviceAccount("test-account")
+        .volumes(volumes)
+        .build();
+    // When & Then
+    assertThatIllegalArgumentException()
+        .isThrownBy(() -> cronJobHandler.get(config, images))
+        .withMessageStartingWith("Invalid upper case letter 'T'")
+        .withMessageEndingWith("controller name value: TesTing");
+  }
+
+  @Test
+  void get_withoutControllerName_shouldThrowException() {
+    // Given
+    ResourceConfig config = ResourceConfig.builder()
+        .schedule("* * * * *")
+        .imagePullPolicy("IfNotPresent")
+        .serviceAccount("test-account")
+        .volumes(volumes)
+        .build();
+    // When & Then
+    assertThatIllegalArgumentException()
+        .isThrownBy(() -> cronJobHandler.get(config, images))
+        .withMessage("No controller name is specified!");
+  }
+
+  @Test
+  void overrideReplicas() {
+    // Given
+    KubernetesListBuilder builder = new KubernetesListBuilder().addToItems(new CronJob());
+    // When
+    cronJobHandler.overrideReplicas(builder, 1337);
+    // Then
+    assertThat(builder.buildItems())
+        .singleElement()
+        .isEqualTo(new CronJob());
```

Comment:
This test doesn't seem to be testing anything. It's better if we remove this test.